tsdoc/lib/index.js

```diff
@@ -0,0 +1,2 @@
+console.log('Hello, world!');
+//# sourceMappingURL=index.js.map
```

Comment:
Looks like this repo is missing a .gitignore.

Reply:
Looks like actually the gitignore is the VS/.NET gitignore.

---
In reply to: [190759791](https://github.com/Microsoft/tsdoc/pull/18#discussion_r190759791) [](ancestors = 190759791)